src/decode.c

```diff
@@ -394,25 +394,23 @@ DBB* dbrew_decode(Rewriter* r, uint64_t f)
                 attachPassthrough(ii, PS_66, OE_RM, SC_None, 0x0F, 0x2E, -1);
                 break;
 
-            case 0x40: // cmovo   r,r/m 32/64
-            case 0x41: // cmovno  r,r/m 32/64
-            case 0x42: // cmovc   r,r/m 32/64
-            case 0x43: // cmovnc  r,r/m 32/64
-            case 0x44: // cmovz   r,r/m 32/64
-            case 0x45: // cmovnz  r,r/m 32/64
-            case 0x48: // cmovs   r,r/m 32/64
-            case 0x49: // cmovns  r,r/m 32/64
-                switch(opc2) {
-                case 0x40: it = IT_CMOVO; break;
-                case 0x41: it = IT_CMOVNO; break;
-                case 0x42: it = IT_CMOVC; break;
-                case 0x43: it = IT_CMOVNC; break;
-                case 0x44: it = IT_CMOVZ; break;
-                case 0x45: it = IT_CMOVNZ; break;
-                case 0x48: it = IT_CMOVS; break;
-                case 0x49: it = IT_CMOVNS; break;
-                default: assert(0);
-                }
+            case 0x40:
+            case 0x41:
+            case 0x42:
+            case 0x43:
+            case 0x44:
+            case 0x45:
+            case 0x46:
+            case 0x47:
+            case 0x48:
+            case 0x49:
+            case 0x4A:
+            case 0x4B:
+            case 0x4C:
+            case 0x4D:
+            case 0x4E:
+            case 0x4F:
+                it = IT_CMOVO + (opc2 & 0xf);
```

Comment:
Please do not create a dependance between IT_* enum and opcode numbers, this just asks for bugs.
Just leave it as before, and extend it by the new cases. And please keep/add mnenomics comments.